build/make-initrd.sh

```diff
@@ -39,7 +38,10 @@ do
 	ln -sf /sbin/iptables /tmp/hyperstart-rootfs/${bin}
 done
 
-ldd /tmp/hyperstart-rootfs/init | while read line
+LDD_BINARIES=(/init /sbin/ipvsadm /sbin/iptables)
+for bin in ${LDD_BINARIES[@]}
```

Comment:
I did this because I meet some inconsistent issue when using libc(libc-2.23.so) on my host. Since All of them will be carried by hyperstart, it should be fine.
Just please include `pthread.so`, it's needed by ipvsadm.